deb/debian-buster/Dockerfile.aarch64

```diff
@@ -6,7 +6,8 @@ RUN sed -ri "s/(httpredir|deb).debian.org/$APT_MIRROR/g" /etc/apt/sources.list
 
 RUN apt-get update && apt-get install -y apparmor bash-completion btrfs-tools build-essential cmake curl ca-certificates debhelper dh-apparmor dh-systemd git libapparmor-dev libdevmapper-dev libltdl-dev libseccomp-dev pkg-config vim-common libsystemd-dev gnupg dirmngr --no-install-recommends && rm -rf /var/lib/apt/lists/*
 
-ENV GO_VERSION 1.10.2
+ARG GO_VERSION
+ENV GO_VERSION $GO_VERSION
```

Comment:
The build arg can used through the entire build process, but won't be available in the image that gets built. If we don' t need the Go version after building then we don't need to define an `ENV`.